cfme/tests/configure/test_start_page.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+import fauxfactory
+
+from cfme.base.credential import Credential
+from cfme.configure.settings import Visual
+from cfme.configure.access_control import Group, User
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+
+faux_value = fauxfactory.gen_alphanumeric()
+pytestmark = pytest.mark.usefixtures('browser')
+
+
+def get_landing_page():
+    view = navigate_to(Visual, 'All')
+    landing_pages = view.visualstartpage.show_at_login.all_options
+    start_page = []
+    for landing_page in landing_pages:
+        start_page.append(landing_page.text)
+    return start_page
+
+
+@pytest.yield_fixture(scope="module")
+def rbac_user(appliance, role):
+    role = role
+    group = Group(description='test{}'.format(faux_value), role=role.name)
+    group.create()
+    group_user = Group('test{}'.format(faux_value))
+    username = 'test{}'.format(faux_value.lower())
+    password = 'test'
+    cred = Credential(principal=username, secret=password)
+    user = User(name='test{}'.format(faux_value),
+                credential=cred, group=group_user)
+    user.create()
+    login_page = navigate_to(appliance.server, 'LoginScreen')
+    login_page.log_in(user)
+    view = navigate_to(Visual, 'All')
+    landing_pages = view.visualstartpage.show_at_login.all_options
+    start_page = []
+    for landing_page in landing_pages:
+        start_page.append(landing_page.text)
+    yield user, landing_pages
+    login_page = navigate_to(appliance.server, 'LoginScreen')
+    login_page.login_admin()
+    user.delete()
+    group.delete()
+
+
+def set_landing_page(value, appliance, user=None):
+    # This list contains the list of pages which show some error or alerts after login.
+    # TODO remove all these pages when BZ is closed.
+    page_list = ['Bottlenecks', 'Automate Log', 'Compute / Containers / Containers']
+    view = navigate_to(Visual, 'All')
+    if (view.visualstartpage.show_at_login.fill(value) and
```

Comment:
**Required**
Switch the order of these statements in the `if`, so that the test doesn't spend time filling forms that it won't save anyway because the page is excluded.